openid-sharedsignals-framework-1_0.md

```diff
@@ -798,10 +798,24 @@ events_delivered
 
 delivery
 
-> **Receiver-Supplied**, A JSON object containing a set of name/value pairs
-  specifying configuration parameters for the SET delivery method.  The actual
-  delivery method is identified by the special key "method" with the value being
-  a URI as defined in {{delivery-meta}}.
+> A JSON object containing a set of name/value pairs specifying configuration
+  parameters for the SET delivery method. The actual delivery method is
+  identified by the special key "method" with the value being a URI as defined
+  in {{delivery-meta}}. The value of the "delivery" field contains two
+  sub-fields: 
+  
+>   delivery_method
+
+> > **Receiver-Supplied**, the specific delivery method to be used. This can be
+    any one of "push" (as defined in {{delivery-push}}) or "poll" (as defined
+    in {{delivery-poll}}), but not both.
```

Comment:
Should we align on the convention and use URN instead of "push" vs "pull" as per #49  changes?